pom.xml

```diff
@@ -1938,7 +1910,8 @@
         <scala.version>${scala12.version}</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
         <hudi.spark.module>hudi-spark3.2.x</hudi.spark.module>
-        <hudi.spark.common.module>hudi-spark3-common</hudi.spark.common.module>
+        <!-- This glob has to include hudi-spark3-common, hudi-spark3.2plus-common -->
+        <hudi.spark.common.modules.glob>hudi-spark3*-common</hudi.spark.common.modules.glob>
```

Comment:
Does it make sense to avoid glob pattern and define separate parameter like `hudi.spark2.common.module`, `hudi.spark3.common.module`, `hudi.spark32plus.common.module`, `hudi.spark33plus.common.module` (in future)? It makes it easier to cherry-pick, otherwise we need to maintain glob pattern every so often.

Reply:
Yeah, not a big fan of globbing (it's quite brittle). Let me try to have separate module (the only reason i opted for globbing initially was b/c wasn't sure if Maven will be able to handle an empty clause, since for ex, this parameter for Spark 2 and 3.1 would be empty)

Reply:
+1 prefer to explicitly add